sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/httpSender.ts

```diff
@@ -27,16 +30,30 @@ export class HttpSender implements Sender {
     this._appInsightsClient = new ApplicationInsightsClient({
       ...this._appInsightsClientOptions
     });
+
+    this._appInsightsClient.pipeline.removePolicy({ name: redirectPolicyName });
```

Comment:
are you sure you're handling redirect responses yourself in all operations?

Reply:
Hi Deyaa, talked to @hectorhdzg that our service only needs to handle 307 and 308, and we have those properly handled in trace.ts [here](https://github.com/Azure/azure-sdk-for-js/blob/a1b6df4ab4ca15b401e649537f2cf4d97a0a5709/sdk/monitor/monitor-opentelemetry-exporter/src/export/trace.ts#L127-L128).

Reply:
I see. And to confirm `exportEnvelopes` is called on each response you get from each operation you expose?

Reply:
i.e. does it act as a "pipeline policy"?

Reply:
and how about the other redirect codes? do not you want to be able to handle them? perhaps have a redirect policy that handle the remaining ones (not 307 and 308).

Reply:
talked offline.